addons/web/controllers/pivot.py

```diff
@@ -71,6 +74,8 @@ def export_xls(self, data, token):
                     worksheet.write(y, x+i, '', header_plain)
                 x = x + (2 * origin_count - 1)
             x, y = 1, y + 1
+            # set minimum width to cells to 20 which around 88px
```

Comment:
comment still makes no sense

Reply:
@aab-odoo Sorry, I did change in hurry, I fixed it now